addons/account/account_invoice.py

```diff
@@ -806,7 +806,8 @@ def action_move_create(self):
             if inv.move_id:
                 continue
 
-            ctx = dict(self._context, lang=inv.partner_id.lang)
+            ctx = dict(self._context, lang=inv.partner_id.lang,
+                       novalidate=True)
```

Comment:
I remember a similar change applied for Saber&Sabor from odoo 5.0

After apply this fix we had another ones related but I'm not remember them.
@JulioSerna Do you remember the problems?

So, I'm scared to apply this change for global way.

However, It performance issue was too much time ago and [current saas-18 don't have this flag enabled](https://github.com/Vauxoo/odoo/blob/160f065251531cafc2066f6a63457152204e2908/addons/account/models/account_invoice.py#L1068)
So, maybe we could create a new PR to odoo/odoo with OPW in order to manage it.

Voy a explicar el problema en español que tuvimos en Saber&Sabor.
1. Cada línea nueva que se creaba (account.move.line) va y hace un proceso de validación del movimiento (account.move) leyendo todas las líneas.
Ente proceso lo hace exponencial, porque, hace lo siguiente:
1. Creo línea 1, checa 1 línea.
2. Creo línea 2, checa línea 1 y línea 2.
3. Creo línea 3, checa línea 1, línea 2 y línea 3.
...
N. Creo línea N, checa línea 1, línea 2... y línea N.

Reply:
cc @nhomar